terraform.tfvars

```diff
@@ -4,17 +4,17 @@
 
 # Ubuntu Image, check latest on https://docs.outscale.com/en/userguide/Official-OMIs-Reference.html
 image_id                  = "ami-9b41b457" # Ubuntu-22.04-2023.02.21-0 (created: 2023-02-21) on us-east-2
-control_plane_vm_type     = "tinav5.c4r8p1"
+control_plane_vm_type     = "tinav5.c2r2p1"
 control_plane_count       = 1
 control_plane_volume_type = "io1"
 control_plane_volume_size = 15
 control_plane_iops        = 1500
-worker_vm_type            = "tinav5.c4r8p1"
+worker_vm_type            = "tinav5.c2r2p1"
 worker_count              = 2
 worker_volume_type        = "io1"
 worker_volume_size        = 15
 worker_iops               = 1500
-bastion_vm_type           = "tinav5.c4r8p1"
+bastion_vm_type           = "tinav5.c1r1p1"
```

Comment:
Really to not small.
https://www.suse.com/support/kb/doc/?id=000020105
Resource needs can vary based on cluster size and workload, however, in general, no less than 8GB of memory and 2 vCPUs is recommended

Reply:
tinav6.c2r2p1

Reply:
fixed